docs/README.md

```diff
@@ -427,25 +427,27 @@ public handle(register: Register): void {
 ```
 
 We can do any validation in the command handler before storing the event, for our
-example, we'll just save the received data in a `PostCreated` event.
+example, we'll just save the received data in the `PostCreated` event.
 
 #### 4. First entity
 
 So far, our `PostCreated` event suggests we need a `Post` entity. Entities are a
-representation of the current state, so an Entity reduces all the events with the same
-`entityID`. Let's then use the entities generator:
+representation of our system internal state. They are in charge of reducing (combining) all the events
+with the same `entityID`. Let's generate our `Post` entity:
 
 ```bash
 boost new:entity Post --fields title:string content:string author:string --reduces PostCreated
 ```
 
-This time Booster has created a file called `Post.ts` in the `src/entities` directory.
+You should see now a new file called `Post.ts` in the `src/entities` directory.
 
-The generator creates one reducer function for each kind of event. As we only have one
-event yet, it will create one function. The reducer functions in Booster work similarly to
-the `reduce` callback functions in Node: they receive an event and the previous state and
-generate a new version of the state. When we receive a `PostCreated` event, we just return
-a new `Post` copying the fields from the event:
+This time, besides using the `--fields` flag to indicate the properties of our `Post` entity, we have
+also used the `--reduces` flag to specify the events the entity will reduce to produce the Post current
+state. The generator will create one _reducer function_ for each event we have specified (only one in this case).
```

Comment:
```suggestion
This time, we use the `--fields` flag to indicate the Post' entity properties. There is also a `--reduces` flag to specify the events the entity will reduce, and therefore, produce the Post current state. The generator will create one _reducer function_ for each event we have specified (only one in this case).
```

Reply:
Just a side note about the docs: Can we configure the markdown linter to have paragraphs in a single line? That really helps to contribute at review time...

Reply:
Yes, done https://github.com/boostercloud/booster/pull/249

Reply:
Done with a smale change here: d0f8b1b